numpy/fft/_pocketfft.py

```diff
@@ -79,13 +75,42 @@ def _raw_fft(a, n, axis, is_real, is_forward, inv_norm):
     return r
 
 
-def _unitary(norm):
-    if norm is None:
-        return False
-    if norm=="ortho":
-        return True
-    raise ValueError("Invalid norm value %s, should be None or \"ortho\"."
-                     % norm)
+def _go_forw(n, norm):
```

Comment:
Maybe rename to `_get_forward_normalization`?  I guess these functions are so short that any docstring is not helpful :).

Reply:
ok